src/serialmonitor/serialportctrl.ts

```diff
@@ -124,7 +101,7 @@ export class SerialPortCtrl {
         resolve();
         return;
       }
-      this._child.stdin.write(`{"cmd": "write", "payload": "${text}"}\n`, (error) => {
+      this._port.write(text + "\n", (error) => {
```

Comment:
This is the same as what we were previously writing during sendMessage, but what purpose does adding a newline serve?

Reply:
This [matches existing behavior from serial-monitor-cli](https://github.com/microsoft/serial-monitor-cli/blob/5bc2764b76a0960ab8c22a6a6ee4890ba52fdf59/src/serialMonitorCtrl.py#L71). While the serial-monitor-cli end of line character is technically configurable [using the `-e` command line argument](https://github.com/microsoft/serial-monitor-cli/blob/5bc2764b76a0960ab8c22a6a6ee4890ba52fdf59/src/main.py#L27-L35), [vscode-arduino never specifies that argument](https://github.com/microsoft/vscode-arduino/blob/08eabe6d845c024dd243b058f54a9f32b73e5296/src/serialmonitor/serialportctrl.ts#L98-L99) so it will always have its default value of `\n`.

Eventually newline should be configurable to address #1324.